### ISSUE TITLE
SET MAC and simulator HELP

### ISSUE THREAD

**Issue by afberendsen** [open-simh/simh#221]
Hi

After some code hunting, I realised that the correct syntax for `SET device MAC`  is
```
SET device MAC=mac-address[/fixed-bits][>conf-file]
```

Maybe the HELP for all simulators should show the full command syntax.

**Comment by afberendsen** [comment]
This is true for all SimH simulators which are using the code from `eth_get_nic_hw_addr' @  `simh-4.1/sim_ether.c`'. This is not limited to the PDP/AX series of simulators.

**Comment by rcornwell** [comment]
Yes I am aware. B5500, IBM 7000, PDP10 simulators are in what I call mature status. This means high issue bugs are handled right away. Low priority bugs and enhancements are handle as time permits. Redoing help system and expanding the help messages is an enhancement. If you wish to collect help message update into one issue, when I get some time I will put them into help system.
